application/src/main/java/org/togetherjava/tjbot/commands/mathcommands/wolframalpha/misunderstoodqueries/Tips.java

```diff
@@ -0,0 +1,42 @@
+package org.togetherjava.tjbot.commands.mathcommands.wolframalpha.misunderstoodqueries;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonRootName;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+@SuppressWarnings("unused")
```

Comment:
remove this suppression. if u intend to not use it - remove the class all together...